resources/main_test.go

```diff
@@ -0,0 +1,132 @@
+package resources
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/terraform"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+var debugFlag = false
+
+func TestMain(m *testing.M) {
```

Comment:
I think it is not necessary to use `TestMain` here, since we already have the helper function `ResourceTypeTest` and could do the set up and tear down there.

We might want to add a test which doesn't need Terraform into this package.